frontend/src/__tests__/integration/hooks/useFetchState.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import useFetchState from '~/utilities/useFetchState';
+import { TestHook } from './TestHook';
+
+export default {
+  component: TestHook,
+  parameters: {
+    a11y: {
+      disable: true,
+    },
+  },
+} as Meta<typeof TestHook>;
+
+const Template: StoryFn<typeof TestHook> = (args) => <TestHook {...args} />;
+
+export const Success = {
+  render: Template,
+  args: {
+    hook: useFetchState,
+    defaultHookParams: [() => Promise.resolve('success-test-state'), 'default-test-state'],
+  },
+};
+
+export const Failure = {
+  render: Template,
+  args: {
+    hook: useFetchState,
+    defaultHookParams: [() => Promise.reject('error-test-state'), 'default-test-state'],
+  },
+};
+
+export const Stable = {
+  render: Template,
+  args: {
+    hook: useFetchState,
+    defaultHookParams: [() => Promise.resolve([1, 2, 3]), []],
+    hookParams: [
+      [() => Promise.resolve([1, 2, 4]), []],
+      [() => Promise.resolve([1, 2, 4]), []],
```

Comment:
Any reason you get 2 same params here?

Reply:
its there so you can test changing the array object when the elements of both array are the same... to see how it updates